### DISCUSSION TITLE
It’s not recommended to suspend a render based on a store value returned by useSyncExternalStore.

### DISCUSSION THREAD

**Discussion by XDUyhfu** [pmndrs/zustand#2282] [Q&A]
![image](https://github.com/pmndrs/zustand/assets/49051362/ef607e40-4c4e-46e7-b9e6-ed789d4a46b4)

Will I also encounter this problem if I use zustand?

**Comment by dai-shi** [comment]
Yes. It's not recommended but it works.
[use-zustand](https://github.com/dai-shi/use-zustand) is a workaround.

See also: https://github.com/facebook/react/issues/26814
